### ISSUE TITLE
Missing variants

### ISSUE THREAD

**Issue by unknownTH** [frappe/erpnext_shopify#146]
Hi, 

I am currently testing ERPNext. Some of my synchronized variants get overwritten with another product variant and are missing in ERPNext. I noticed that the “item_code” has usually the same value as the “shopify_product_id”. The overwritten variants have two different values in the mentioned fields. If you want I can provide you an export of my ERPNext items.

It would be great if you could help. 

Best regards

**Comment by me2ne** [comment]
you are right , in shopify there is sky and should sync with item code in erpnext
